@jechter  I'm getting a compile error on this branch.

`Editor\AssetEditor\InputActionEditorWindow.cs(510,27): error CS0122: 'InputActionTreeView.UpdateSerializedObjectDirtyCount()' is inaccessible due to its protection level`

--- a/Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionTreeView.cs
+++ b/Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionTreeView.cs
@@ -1192,7 +1192,7 @@ namespace UnityEngine.Experimental.Input.Editor
             onSerializedObjectModified?.Invoke();
         }
 
-        private void UpdateSerializedObjectDirtyCount()
+        public void UpdateSerializedObjectDirtyCount()
         {
             m_SerializedObjectDirtyCount = EditorUtility.GetDirtyCount(serializedObject.targetObject);
         }

